manifests/user.pp

```diff
@@ -55,6 +55,10 @@
     })
   }
 
+  ensure_resource('group', $name, {
```

Comment:
Would you mind moving the other `ensure_resource('group'` block at the top of the file is either brought down. Or move this `ensure_resource('group'` and `ensure_resource('user'` up to the top please?

Otherwise looks good!

Thanks!

Reply:
Done